src/platform/pret_sdl/sdl2.c

```diff
@@ -551,7 +551,7 @@ static void RunDMAs(u32 type)
             if (dma->control & DMA_REPEAT) {
                 dma->size = ((&REG_DMA0CNT)[dmaNum * 3] & 0x1FFFF);
                 if (((dma->control) & DMA_DEST_MASK) == DMA_DEST_RELOAD) {
-                    dma->dst = (void *)((&REG_DMA0DAD)[dmaNum * 3]);
+                    dma->dst = *(void **)&((&REG_DMA0DAD)[dmaNum * 3]);
```

Comment:
What's the reason for changing this?

Reply:
it's just a way to force the value to a pointer. The value is `u32` so can't be cast to `void *` in 64bit